BrowserServicesKit/Tests/SubscriptionTests/Managers/SubscriptionManagerTests+Redirect.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  SubscriptionManagerTests+Redirect.swift
+//  BrowserServicesKit
+//
+//  Created by Michał Smaga on 18/02/2025.
+//
+
+import XCTest
+import Common
+@testable import Subscription
+import SubscriptionTestingUtilities
+
+extension SubscriptionManagerTests {
```

Comment:
You can use these tests as examples how resolving the purchase URL based on redirect works (similar test cases are listed in the description).